projects/angular/src/timeline/_timeline.clarity.scss

```diff
@@ -93,6 +92,10 @@
   .clr-timeline-step-body {
     display: flex;
     flex-direction: column;
+    cds-icon,
```

Comment:
Is there a reason for including this? cds-icons and I believe clr-icons have a default size included.